### ISSUE TITLE
pip or conda version

### ISSUE THREAD

**Issue by mayou36** [madebr/pyOpt#13]
Hi all, are there any plans for a release on pip or conda-forge?

**Comment by madebr** [comment]
Not currently, but it would be nice to have a github workflow to create a new pypy release when creating a git tag.

**Comment by mayou36** [comment]
Okay, I think the library has a lot of potential. Since it is your fork, what are your plans and willingness to maybe change and improve things? I am glad to contribute something, but it requires of course also readiness to accept PRs etc.

**Comment by madebr** [comment]
I'm willing to accept anything as long as it's within scope.
Having a rudimentary CI should prove that the code is working.

I think the 
- Use https://github.com/actions/setup-python to build PyOpt for all relevant python versions
- On Ubuntu, use gfortran from the package manager
- On Windows, use fortran from msys2 (https://github.com/msys2/setup-msys2)
Windows is a stretch goal, imho.
- Some care should be taken to ensure PyOpt runs without any fortran runtime libraries.
- Use https://github.com/pypa/gh-action-pypi-publish to publish the package.

I think this will make it possible to easily test + publish PyOpt.

**Comment by mayou36** [comment]
I agree to the CI and github actions, I would however opt for the conda compilers for the testing as this are available everywhere and may allows for a deployment with conda-forge at some point

> Some care should be taken to ensure PyOpt runs without any fortran runtime libraries.

Why this point? Like not to force the usage of fortran?
